src/community/ogcapi/ogcapi-tiles/src/main/java/org/geoserver/ogcapi/tiles/TilesService.java

```diff
@@ -52,6 +52,7 @@
 import org.geowebcache.io.ByteArrayResource;
 import org.geowebcache.io.Resource;
 import org.geowebcache.layer.TileLayer;
+import org.geowebcache.layer.meta.TileJSON;
```

Comment:
This feels like it should be split out - its vaguely related, but its really a different change.